hope.yaml

```diff
@@ -10,15 +10,88 @@ _:
   - &plex_load_balancer_service_ip LOAD_BALANCER_IP=192.168.200.69
   - &pihole_ingress_controller_ip INGRESS_IP=192.168.200.128
   - &unifi_controller_service_ip LOAD_BALANCER_IP=192.168.200.82
-master_load_balancer: api.internal.aleemhaji.com
-masters:
-  - root@192.168.200.10
-  # - root@192.168.1.31
-  # - root@192.168.1.80
+vms:
+  root: vms
+  cache: /var/lib/packer/cache
+  output: /var/lib/packer/images
+  images:
+    - name: load-balancer
+      parameters:
+        - ESXI_ROOT_PASSWORD
+        - SERVER_SSH_KEYS_DIR=/var/lib/packer/etc/ssh
+        - ESXI_NETWORK=VM Network
+        - VM_ROOT_PASSWORD
+        - VM_MANAGEMENT_PASSWORD
+    - name: kubernetes-node
+      parameters:
+        - ESXI_ROOT_PASSWORD
+        - SERVER_SSH_KEYS_DIR=/var/lib/packer/etc/ssh
+        - ESXI_NETWORK=VM Network
+        - VM_ROOT_PASSWORD
+        - VM_MANAGEMENT_PASSWORD
+load_balancer_host: api.internal.aleemhaji.com
 nodes:
-  # - root@192.168.1.30
-  - ubuntu@192.168.200.10
-  - ubuntu@192.168.200.11
+  # Old style; explicit hosts, no hypervisor
+  # - name: util1
```

Comment:
Kept these around so I can still manage them during the migration, but should be eliminated soon.